lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py

```diff
@@ -75,6 +75,11 @@
         description:
             - List of IpTag associated with the public IP address.
             - Each element should contain type:value pair.
+        suboptions:
+            type: 
+                description: Sets the ip_tags type: Example FirstPartyUsage.
```

Comment:
This is an invalid yaml.

We usually write the description as a list. BTW, there is no need to add the example. if there is ":" in the sentence, the whole line should be quoted.

```yml
description:
    - "XXXXX"
```